index.html

```diff
@@ -35,7 +40,11 @@
       </nav>
     </header>
     <section class="headline container">
+<<<<<<< HEAD
+      <h1 class="primary-header">I’m Emmanuel Kipngeno. Glad to see you!</h1>
+=======
       <h1 class="primary-header">I’m Emmanuel kipngeno. Glad to see you!</h1>
+>>>>>>> 37af104b863833cd5ed42ca5ead3e1ebb99ebda7
```

Comment:
- [ ] These might be merge conflict and I think you should need to fix them because they are breaking your design 

![image](https://user-images.githubusercontent.com/75126481/235323188-b02c5171-fbf2-4ec0-9405-98df1b2b7f46.png)